vector/src/main/java/im/vector/app/features/home/room/detail/timeline/action/MessageActionsViewModel.kt

```diff
@@ -355,11 +356,15 @@ class MessageActionsViewModel @AssistedInject constructor(@Assisted
                 add(EventSharedAction.Copy(messageContent!!.body))
             }
 
-            if (timelineEvent.canReact() && actionPermissions.canReact) {
+            // Tchap: Disable message reaction
+            if (timelineEvent.canReact() && actionPermissions.canReact &&
+                    BuildConfig.SHOW_ADD_MESSAGE_REACTION) {
                 add(EventSharedAction.AddReaction(eventId))
             }
 
-            if (canViewReactions(timelineEvent)) {
+            // Tchap: Disable message reaction
+            if (canViewReactions(timelineEvent) &&
+                    BuildConfig.SHOW_ADD_MESSAGE_REACTION) {
```

Comment:
Please revert this change as it is only displayed to see the current reactions, not to add new ones